app/code/core/Mage/Catalog/etc/system.xml

```diff
@@ -462,14 +462,6 @@
                     <show_in_website>1</show_in_website>
                     <show_in_store>1</show_in_store>
                     <fields>
-                        <size translate="label">
-                            <label>Watermark Default Size</label>
-                            <sort_order>100</sort_order>
-                            <show_in_default>1</show_in_default>
-                            <show_in_website>1</show_in_website>
-                            <show_in_store>1</show_in_store>
-                            <comment>Example format: 200x300.</comment>
```

Comment:
what I would find more useful (but I wouldn't care if it's missing now, it a percentage of the size of the image that the watermark is applied to) because this way it auto adapts to various source image sizes

Reply:
I was also thinking we could have options for cover and contain, same as CSS [background-size](https://developer.mozilla.org/en-US/docs/Web/CSS/background-size).

Actually, there should be at least two fields: position (top, left, etc) and size (cover, contain, stretch, tile, percentage). Right now they're combined  into one dropdown which doesn't give as much flexibility, so that's definitely a larger effort.